src/test/java/com/wikia/webdriver/TestCases/adstests/TestEvolveAds.java

```diff
@@ -0,0 +1,65 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.geoedge.GeoEdgeProxy;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsEvolveObject;
+import org.testng.annotations.Test;
+
+/**
+ * @author Dmytro Rets
+ * @ownership AdEngineering
+ */
+public class TestEvolveAds extends NewTestTemplate {
+
+	private static final String EVOLVE_ERROR_MESSAGE = "Verify Evolve call in ";
+
+	@GeoEdgeProxy(country = "CA")
+	@Test(
+		dataProviderClass = AdsDataProvider.class,
+		groups = {"TestEvolveAds"},
+		dataProvider = "evolveTestPage"
+	)
+	public void testEvolveCall_CA(String wikiName, String article) {
+		testEvolveCall(wikiName, article);
+	}
+
+	@GeoEdgeProxy(country = "AU")
+	@Test(
+		dataProviderClass = AdsDataProvider.class,
+		groups = {"TestEvolveAds"},
+		dataProvider = "evolveTestPage"
+	)
+	public void testEvolveCall_AU(String wikiName, String article) {
+		testEvolveCall(wikiName, article);
+	}
+
+	@GeoEdgeProxy(country = "NZ")
+	@Test(
+		dataProviderClass = AdsDataProvider.class,
+		groups = {"TestEvolveAds"},
+		dataProvider = "evolveTestPage"
+	)
+	public void testEvolveCall_NZ(String wikiName, String article) {
+		testEvolveCall(wikiName, article);
+	}
+
+	private void testEvolveCall(String wikiName, String article) {
```

Comment:
Please move the implementations of the test into the page object class such as AdsEvolveObject.